.github/workflows/flake8-to-ruff.yaml

```diff
@@ -34,7 +34,6 @@ jobs:
         with:
           target: x86_64
           args: --release --out dist --sdist -m ./${{ env.CRATE_NAME }}/Cargo.toml
-          maturin-version: "v0.13.0"
```

Comment:
maturin-action by default read maturin version from `pyproject.toml` and install a semver compatible version, see https://github.com/PyO3/maturin-action/pull/46